examples/webgl_animation_multiple.html

```diff
@@ -1,390 +1,335 @@
 			}
 
-			/**
-			 * Find a model object by name
-			 * @param name
-			 * @returns {object|null}
-			 */
-			function getModelByName( name ) {
+			// Different models can have different configurations of armatures and meshes. Therefore,
+			// We can't set position, scale or rotation to individual mesh objects. Instead we set
+			// it to the whole cloned scene and then add the whole scene to the game world
+			// Note: this may have weird effects if you have lights or other items in the GLTF file's scene!
+			worldScene.add(clonedScene);
 
-				for ( let i = 0; i < MODELS.length; ++ i ) {
-
-					if ( MODELS[ i ].name === name ) {
+			if (u.position) {
+				clonedScene.position.set(u.position.x, u.position.y, u.position.z);
+			}
 
-						return MODELS[ i ];
+			if (u.scale) {
+				clonedScene.scale.set(u.scale, u.scale, u.scale);
+			}
 
-					}
+			if (u.rotation) {
+				clonedScene.rotation.x = u.rotation.x;
+				clonedScene.rotation.y = u.rotation.y;
+				clonedScene.rotation.z = u.rotation.z;
+			}
+		}
+
+		/**
+		 * Start animation for a specific mesh object. Find the animation by name in the 3D model's animation array
+		 * @param skinnedMesh {THREE.SkinnedMesh} The mesh to animate
+		 * @param animations {Array} Array containing all the animations for this model
+		 * @param animationName {string} Name of the animation to launch
+		 * @return {THREE.AnimationMixer} Mixer to be used in the render loop
+		 */
+		function startAnimation(skinnedMesh, animations, animationName) {
+			const mixer = new THREE.AnimationMixer(skinnedMesh);
+			const clip = THREE.AnimationClip.findByName(animations, animationName);
+
+			if (clip) {
+				const action = mixer.clipAction(clip);
+				action.play();
+			}
 
+			return mixer;
+		}
+
+		/**
+		 * Find a model object by name
+		 * @param name
+		 * @returns {object|null}
+		 */
+		function getModelByName(name) {
+			for (let i = 0; i < MODELS.length; ++i) {
+				if (MODELS[i].name === name) {
+					return MODELS[i];
 				}
-
-				return null;
-
 			}
 
-			/**
-			 * Load a 3D model from a GLTF file. Use the GLTFLoader.
-			 * @param model {object} Model config, one item from the MODELS array. It will be updated inside the function!
-			 * @param onLoaded {function} A callback function that will be called when the model is loaded
-			 */
-			function loadGltfModel( model, onLoaded ) {
-
-				const loader = new GLTFLoader();
-				const modelName = "models/gltf/" + model.name + ".glb";
+			return null;
+		}
 
-				loader.load( modelName, function ( gltf ) {
+		/**
+		 * Load a 3D model from a GLTF file. Use the GLTFLoader.
+		 * @param model {object} Model config, one item from the MODELS array. It will be updated inside the function!
+		 */
+		function loadGltfModel(url = "") {
+			const loader = new GLTFLoader();
 
-					const scene = gltf.scene;
-
-					model.animations = gltf.animations;
-					model.scene = scene;
+			return new Promise((resolve, reject) => {
+				loader.load(url, function (gltf) {
```

Comment:
You can use `gltfloader.loadAsync()` as demonstrated here instead of wrapping it in a promise.

https://github.com/mrdoob/three.js/blob/9c82abe72bada04a41dda921b2ab8818be69b532/examples/webgl_tonemapping.html#L94